apps/knowte/knowte.yml

```
@@ -0,0 +1,8 @@
+name: Knowte
+description: 'Cross platform note taking application'
+website: 'https://www.digimezzo.com'
+category: Productivity
+repository: 'https://github.com/digimezzo/knowte-electron'
+keywords:
+    - 'note notes text organize search quick'
```

I think you mean these to be separate keywords?

```suggestion
    - note
    - notes
    - text
    - organize
    - search
    - quick
```

---

Yes indeed